### ISSUE TITLE
Recorded file saving failure

### ISSUE THREAD

**Issue by Karetski** [audiokit/AudioKit#918]
I'm trying to add to "Recording Nodes" playground the following code:
```
addSubview(AKButton(title: "Save") {
            tape.exportAsynchronously(name: "test", baseDir: .documents, exportFormat: .caf) { [weak self] exportedFile, error in
            }
            return "Saved"
        })
```
And when i try to save recorded file, i get the following:
```
initialize() AKAudioPlayer Warning:  "6399C87A-6727-4579-80D3-B7B951762216.caf" is an empty file
record() AKNodeRecorder: recording
record() AKNodeRecorder writing (file duration: 0.1 seconds)
record() AKNodeRecorder writing (file duration: 0.2 seconds)
record() AKNodeRecorder writing (file duration: 0.3 seconds)
record() AKNodeRecorder writing (file duration: 0.4 seconds)
record() AKNodeRecorder writing (file duration: 0.5 seconds)
record() AKNodeRecorder writing (file duration: 0.6 seconds)
record() AKNodeRecorder writing (file duration: 0.7 seconds)
record() AKNodeRecorder writing (file duration: 0.8 seconds)
record() AKNodeRecorder writing (file duration: 0.9 seconds)
record() AKNodeRecorder writing (file duration: 1.0 seconds)
record() AKNodeRecorder writing (file duration: 1.1 seconds)
record() AKNodeRecorder writing (file duration: 1.2 seconds)
record() AKNodeRecorder writing (file duration: 1.3 seconds)
record() AKNodeRecorder writing (file duration: 1.4 seconds)
record() AKNodeRecorder writing (file duration: 1.5 seconds)
record() AKNodeRecorder writing (file duration: 1.6 seconds)
record() AKNodeRecorder writing (file duration: 1.7 seconds)
record() AKNodeRecorder writing (file duration: 1.8 seconds)
record() AKNodeRecorder writing (file duration: 1.9 seconds)
record() AKNodeRecorder writing (file duration: 2.0 seconds)
record() AKNodeRecorder writing (file duration: 2.1 seconds)
record() AKNodeRecorder writing (file duration: 2.2 seconds)
record() AKNodeRecorder writing (file duration: 2.3 seconds)
record() AKNodeRecorder writing (file duration: 2.4 seconds)
record() AKNodeRecorder writing (file duration: 2.5 seconds)
record() AKNodeRecorder writing (file duration: 2.6 seconds)
record() AKNodeRecorder writing (file duration: 2.7 seconds)
record() AKNodeRecorder writing (file duration: 2.8 seconds)
record() AKNodeRecorder writing (file duration: 2.9 seconds)
record() AKNodeRecorder writing (file duration: 3.0 seconds)
record() AKNodeRecorder writing (file duration: 3.1 seconds)
record() AKNodeRecorder writing (file duration: 3.2 seconds)
record() AKNodeRecorder writing (file duration: 3.3 seconds)
record() AKNodeRecorder writing (file duration: 3.4 seconds)
record() AKNodeRecorder writing (file duration: 3.5 seconds)
record() AKNodeRecorder writing (file duration: 3.6 seconds)
record() AKNodeRecorder writing (file duration: 3.7 seconds)
record() AKNodeRecorder writing (file duration: 3.8 seconds)
record() AKNodeRecorder writing (file duration: 3.9 seconds)
record() AKNodeRecorder writing (file duration: 4.0 seconds)
record() AKNodeRecorder writing (file duration: 4.1 seconds)
record() AKNodeRecorder writing (file duration: 4.2 seconds)
record() AKNodeRecorder writing (file duration: 4.3 seconds)
record() AKNodeRecorder writing (file duration: 4.4 seconds)
record() AKNodeRecorder writing (file duration: 4.5 seconds)
record() AKNodeRecorder writing (file duration: 4.6 seconds)
updatePCMBuffer() read 202860 frames into buffer
exportAsynchronously(name:baseDir:exportFormat:fromSample:toSample:callback:) internalExportSession session created
exportAsynchronously(name:baseDir:exportFormat:fromSample:toSample:callback:) ERROR AKAudioFile export: In time must be less than Out time!...
queueExportSession(session:) ExportFactory: exporting session #0
completionHandler() ExportFactory: session #0 Completed
completionHandler() ExportFactory: All exports have been completed
```
Same issue reproduced on iOS Simulator

**Comment by whisper-bye** [comment]
+1 pods need a bug-fix release #849
https://github.com/audiokit/AudioKit/commit/724d14cb77183dc57e8c47e5afca8e0163ed807f

**Comment by aure** [comment]
This works in AudioKit 3.7 playgrounds.  I'll add the save button idea to the playgrounds in next release.
